app/templates/editor/common/edit.pug

```diff
@@ -0,0 +1,29 @@
+extends /templates/base
+
+block content
+  div
+    ol.breadcrumb
+      li
+        a(href="/editor", data-i18n="editor.main_title") CodeCombat Editors
+      li
```

Comment:
<img width="1970" alt="Screen Shot 2022-06-15 at 17 28 34" src="https://user-images.githubusercontent.com/99704/173964403-eabd57e2-6885-4650-810d-2b767765a6c2.png">

Love the start at making these more general. Noticed a few things to update:
1. resource name isn't appearing on general editor page
2. Create a New Something instead of Create a New Podcast
3. Search table should default to linking to slug URLs instead of _id URLs
4. "podcast Editor" instead of "Podcast Editor"
5. alignment, order, and margin of Populate i18n and Save buttons